src/classes/RD_RecurringDonations.cls

```diff
@@ -30,6 +30,10 @@
 /**
 * @author Salesforce.com Foundation
 * @date 2012 (2.0)
+*
```

Comment:
I've been not leaving any empty lines here, I think we should follow that for consistency.